x-pack/plugins/security_solution/scripts/endpoint/resolver_generator.ts

```diff
@@ -95,19 +95,25 @@ async function main() {
     eventIndex: {
       alias: 'ei',
       describe: 'index to store events in',
-      default: 'events-endpoint-1',
+      default: 'logs-endpoint.events.fake-default',
```

Comment:
what is fake default?

Reply:
So we're actually splitting up all the `events` into their own indices in the mapping aka `endpoint.events.network` `endpoint.events.library` `endpoint.events.file` etc. But to support that I'd have to overhaul the generator to send each separate type of event to a specific index. As a shortcut for now I'm just sending them all to `endpoint.events.fake` because it'll still match the pattern.

That way we don't need like 8 command line options for all the different event index names.

Reply:
although you bring up a good point. Eventually this won't work because it won't have a mapping defined. Maybe instead I'll just set it to `logs-endpoint.events.file-default` for now.

Reply:
Yes I agree, we do not want anything not delivered through the infrastructure and also not dynamic. We may also want to review the name.